experiments/conductor/branches-all.yaml

```diff
@@ -48,24 +30,7 @@ branches:
       host-name: ""
       notes:
         - No gcloud command found
-      apis-enabled: []
-    - name: accessapproval-approvalrequest
-      local: resource-accessapproval-approvalrequest
-      remote: resource-accessapproval-approvalrequest
-      command: gcloud access-approval requests
```

Comment:
approval requests can be approved, dismissed or invalidated. That is awfully imperative but I can see how you could make a KCC resource from it. Having said that going to go with it for now. Maybe better to wait for an actual CUJ.